# Determination - Deterministic rendering environment for music and art
# Copyright (C) 2024 Liu Hao <whiteaxe@tuta.io>
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3.

# Disables the Krita plugins we don't need to export images since Krita's
# plugins take up a lot of the code size and compilation time.

--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -11,19 +11,19 @@ include_directories(SYSTEM
     ${Vc_INCLUDE_DIR}
 )
 
-add_subdirectory( assistants )
+#add_subdirectory( assistants )
 add_subdirectory( color )
-add_subdirectory( dockers )
-add_subdirectory( extensions )
+#add_subdirectory( dockers )
+#add_subdirectory( extensions )
 add_subdirectory( filters )
 add_subdirectory( flake )
-add_subdirectory( generators )
+#add_subdirectory( generators )
 add_subdirectory( impex )
 add_subdirectory( metadata )
-add_subdirectory( paintops )
-add_subdirectory( tools )
-add_subdirectory( qt )
-
-if (HAVE_PYQT5 AND HAVE_SIP AND HAVE_PYTHONLIBS)
-    add_subdirectory( python )
-endif()
+#add_subdirectory( paintops )
+#add_subdirectory( tools )
+#add_subdirectory( qt )
+#
+#if (HAVE_PYQT5 AND HAVE_SIP AND HAVE_PYTHONLIBS)
+#    add_subdirectory( python )
+#endif()
--- a/plugins/impex/CMakeLists.txt
+++ b/plugins/impex/CMakeLists.txt
@@ -43,7 +43,7 @@ add_subdirectory(psd)
 add_subdirectory(qml)
 add_subdirectory(tga)
 add_subdirectory(heightmap)
-add_subdirectory(brush)
+#add_subdirectory(brush)
 add_subdirectory(spriter)
 add_subdirectory(kra)
 
